changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.13.0
+============
+New Features: 
+
+**Client throttling is supported in Public Client Applications** MSAL will now implement client side throttling to reduce excessive authentication requests sent to the service. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1624)
```

Comment:
```suggestion
**Client throttling is supported in Public Client Applications** MSAL will now implement client side throttling to reduce excessive authentication requests sent to the service: In the case where the Azure AD service replies with an HTTP error implying throttling, MSAL.NET now respects itself the delay imposed by the service by throwing an exception telling the application after which delay/when it will be able to acquire a token again without even attempting to call the service. For details see [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1624)
```